makefile

```diff
@@ -16,12 +16,9 @@ list: # PRIVATE
 # *********************** SETUP ***********************
 
 # Install all 3rd party dependencies.
-install: check-node
+install: check-node check-yarn
 	@echo 'Installing 3rd party dependencies…'
-	@npm install
-	@echo '…done.'
-	@echo 'Removing any unused 3rd party dependencies…'
-	@npm prune
```

Comment:
I wonder whether we should run `yarn install --force` to have parity with the prune behaviour? Seems like a bug that they may decide `yarn install` will remove modules that aren't in the `package.json`

https://github.com/yarnpkg/yarn/issues/696